### PR TITLE
Template updates: 2.12, 2.13 and 2.14

### PR DESCRIPTION
**Pull request by erikrikarddaniel**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/phyloplace/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/phyloplace _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by erikrikarddaniel** [comment]
@nf-core-bot prettier

**Comment by erikrikarddaniel** [comment]
@nf-core-bot fix linting

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,16 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Version 1.0 - [2023-02-15]
-
-Initial release of nf-core/phyloplace, created with the [nf-core](https://nf-co.re/) template.
-
-The pipeline performs phylogenetic placement of nucleotide or amino acid sequences in a reference phylogeny.
+## v1.1.0dev - [date]
```
Comment (danilodileo) [inline CHANGELOG.md:6]: The template update is removing comments. do you want to keep them?

**Review by danilodileo — APPROVED** [review]
LGTM although it looks there are some test that are still hanging.

**Pull request merged**